src/cloudsc_cuf_loki/cloudsc_cuf_loki.config

```diff
@@ -0,0 +1,65 @@
+derived_types = ['TECLDP']
+
+[dic2p]
+NLEV = 137
+
+[default]
+# Specifies the behaviour of auto-expanded routines
+role = 'kernel'
+expand = true  # Automatically expand subroutine calls
+strict = true  # Throw exceptions during dicovery
+
+# Ensure that we are never adding these to the tree, and thus
+# do not attempt to look up the source files for these.
+# TODO: Add type-bound procedure support and adjust scheduler to it
+#disable = ['timer%start', 'timer%end', 'timer%thread_start', 'timer%thread_end',
```

Comment:
I'll clean up all of the unnecessary ones here. Following some Scheduler improvements in Loki we might even be able to trim this further. I'll leave this as a an exercise for the future integration of this with the main `cloudsc_loki` variants.

Reply:
In fact, a little bit of digging suggests that there is some string-specific dependence in the hoist transform used by one the SCC-CUF variants that requires this for a dict-lookup. This should be fixable by fixing the hoist-transformation, so that we can then merge this back into the `cloudsc_loki` directory and share config setups.